shacl12-core/index.html

```diff
@@ -6413,6 +6411,98 @@ <h4>sh:property</h4>
 						although it is reached via different paths in the <a>shapes graph</a>.
 					</p>
 				</section>
+
+				<section id="SomeValueConstraintComponent">
+					<h4>sh:someValue</h4>
+					<p>
+						<code>sh:someValue</code> specifies the condition that at least one <a>value node</a> conforms to the given <a>shape</a>.
+					</p>
+					<p>
+						<span class="component-class">Constraint Component IRI</span>: <code>sh:SomeValueConstraintComponent</code>
+					</p>
+
+					<div class="parameters">Parameters:</div>
+					<table class="term-table">
+						<thead>
+							<tr>
+								<th>Property</th>
+								<th>Summary and Syntax Rules</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr>
+								<td><code>sh:someValue</code></td>
+								<td>
+									The <a>shape</a> that at least one of the <a>value nodes</a> needs to conform to.
+									<span data-syntax-rule="someValue-shape">The values of <code>sh:someValue</code> in a shape must be <a>well-formed</a> <a>shapes</a>.</span>
+								</td>
+							</tr>
+						</tbody>
+					</table>
+					<div class="def def-text">
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="Node">
+							Let <code>$someValue</code> be a <a>value</a> of <code>sh:someValue</code>.
```

Comment:
Does `$someValue` need to be listed in the table of pre-bound SPARQL variables?  SHACL-SPARQL Section 2.3.2 today.

https://github.com/w3c/data-shapes/blob/7d2d4c28864eea174c5f46eba7e0f0b92c890ea9/shacl12-sparql/index.html#L1061-L1062

Reply:
No, this has nothing to do with SPARQL here. I only used $someValue because I think we should do that everywhere. I started actually doing this with $focusNode in the revised pathValues definition, because the name otherwise clashed with the variable focusNode.